x-pack/solutions/security/packages/kbn-cloud-security-posture/public/src/utils/get_finding_colors.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiThemeComputed } from '@elastic/eui';
+import { euiThemeVars } from '@kbn/ui-theme'; // TODO: replace with euiTheme
+import type { VulnSeverity } from '@kbn/cloud-security-posture-common';
+import {
+  VULNERABILITIES_SEVERITY,
+  MISCONFIGURATION_STATUS,
+} from '@kbn/cloud-security-posture-common';
+
+const isBorealis = (euiThemeName: string) => {
+  return euiThemeName?.toLowerCase().includes('borealis');
+};
+
+const isAmsterdam = (euiThemeName: string) => {
+  return euiThemeName?.toLowerCase().includes('amsterdam');
+};
+
+// TODO: replace with severity color palette
```

Comment:
all these todos should go away when we use only the severity palette for both themes